Cypress/cypress/integration/Parallel_Tests/Data_Source/addDataSource.spec.js

```diff
@@ -77,6 +77,41 @@ describe("Testing the addition of data source", () => {
       });
   });
 
+  it("Changing name of existing datasource and validating it", () => {
+    cy.visit("/observability");
+    cy.get("[data-cy=browseDataSourceOptions]").eq(0).click();

```

Comment:
To make sure that we are in data sources tab.
```suggestion
    cy.visit("/observability");
    cy.get("[data-cy='data source']").click();
    cy.get("[data-cy=browseDataSourceOptions]").eq(0).click();

```